articles/virtual-desktop/troubleshoot-azure-ad-connections.md

```diff
@@ -27,6 +27,28 @@ Visit the [Azure Virtual Desktop Tech Community](https://techcommunity.microsoft
 
 If you come across an error saying **Your account is configured to prevent you from using this device. For more information, contact your system administrator**, ensure the user account was given the [Virtual Machine User Login role](../active-directory/devices/howto-vm-sign-in-azure-ad-windows.md#azure-role-not-assigned) on the VMs. 
 
+### I can't sign in even though I'm using the right credentials
+
+If you experience a login problem presented by incorrect user credentials and have verified they are correct and have ruled out other errors verify that Conditional Access policies excludes multifactor authentication requirements on the **Azure Windows VM Sign-in** cloud application. Also verify that you have assigned the **Virtual Machine User Login** RBAC permissions on the VM or Resource Group for each user. Follow the instructions in [Enable multifactor authentication](deploy-azure-ad-joined-vm.md#enabling-mfa-for-azure-ad-joined-vms)
+
+> [!WARNING] 
+> VM sign-ins don't support per-user enabled or enforced Azure AD multifactor authentication. If you try to sign in with multifactor authentication on a VM, then you won't be able to sign in and will receive an error message.
+
+If you are able to access your Azure AD Sign-in logs through Log Analytics you can verify whether MFA is being applied and what Conditional Access policy is triggering the event. The events shown are the non-interactive user login events that are performed at the VM and therefore the IP address will appear to come from the external IP address that your virtual machine accesses Azure AD from. 
+
+You can access your sign-in logs by running the following Kusto query:
```

Comment:
I think this line is now duplicated. One should be removed.

Reply:
Removed the duplicate line. Thanks